e2e/tests-dfx/create.bash

```diff
@@ -337,4 +338,5 @@ teardown() {
   assert_contains 'Reserved cycles limit: 1_000_000_000_000'
   assert_contains 'Wasm memory limit: 1_073_741_824'
   assert_contains 'Freezing threshold: 604_800'
+  assert_contains 'Log visibility: public'
```

Comment:
Can we update an existing test to show that dfx by default creates canister with `controllers` log visibility?